write-fonts/src/write.rs

```diff
@@ -178,12 +195,28 @@ impl Default for TableWriter {
 }
 
 /// The encoded data for a given table, along with info on included offsets
-#[derive(Debug, Default, Clone, Hash, PartialEq, Eq)]
+#[derive(Debug, Default, Clone)] // DO NOT DERIVE MORE TRAITS! we want to ignore name field
```

Comment:
At a glance I would think if you let the name be included the practical outcome is that you can't reuse the bytes of one table as a valid table of some other type? Is that something we actually do? Are there other implications?

Reply:
Correct, we will currently reuse a table as long as the bytes are identical. I do not have any stats on how often this happens in practice, but it is part of the design.